python_src/src/lamp_py/ingestion/convert_gtfs_rt.py

```diff
@@ -355,6 +344,8 @@ def write_table(self, table: pyarrow.table) -> None:
         try:
             s3_prefix = str(self.config_type)
 
+            table = self.detail.transform_for_write(table)
+
```

Comment:
transform pyarrow table before writing to parquet, this could include flatten and/or exploding